nodeup/pkg/bootstrap/install.go

```diff
@@ -165,16 +146,16 @@ func (i *Installation) buildEnvFile() *nodetasks.File {
 		sysconfig += key + "=" + value + "\n"
 	}
 
-	task := &nodetasks.File{
+	task := &nodetasks.InstallFile{File: nodetasks.File{
```

Comment:
A little awkward codewise, but I see the value!

(Would be great if these ended up as interfaces where the File task could implement both, but we have that conflict on Run... we could get rid of Run maybe, but ... not in this PR!)